examples/math.rs

```diff
@@ -20,18 +20,18 @@ use std::iter::Peekable;
 #[allow(non_camel_case_types)]
 #[repr(u16)]
 enum SyntaxKind {
-    WHITESPACE = 0,
+    Whitespace = 0,
+    Add = 1,
+    Subtract = 2,
+    Multiply = 3,
+    Divide = 4,
 
-    ADD,
-    SUB,
-    MUL,
-    DIV,
-
-    NUMBER,
-    ERROR,
-    OPERATION,
-    ROOT,
+    Number = 5,
+    Error = 6,
+    Operation = 7,
+    Root = 8,
 }
```

Comment:
Can we do undo this, the lint is allowed on top of it for a reason, the casing is intentiuonal as these are meant to be used unprefixed mimicking a bag of constants